lib/components/user/monitored-trip/trip-summary-pane.tsx

```diff
@@ -1,61 +1,236 @@
-import { FormattedList, FormattedMessage } from 'react-intl'
+import { Bell, BellSlash, Calendar, Clock } from '@styled-icons/fa-regular'
+import { FormattedMessage, useIntl } from 'react-intl'
 import React from 'react'
 
 import { dayFieldsToArray } from '../../../util/monitored-trip'
 import { MonitoredTripProps } from '../types'
-import FormattedDayOfWeek from '../../util/formatted-day-of-week'
-import Strong from '../../util/strong-text'
+import LocationIcon from '@opentripplanner/location-icon'
+import styled from 'styled-components'
 
-import TripSummary from './trip-summary'
+import { InlineLoading } from '../../narrative/loading'
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+
+import MonitoredDays from './trip-monitored-days'
+import TripSummary from './trip-duration-summary'
 
 /**
  * Displays the summary information of a monitored trip.
  */
+
+const SavedTripBody = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 0 0 0 15px;
+
+  @media (max-width: 768px) {
+    flex-direction: column;
+    padding: 0;
+  }
+`
+
+const LocationDetails = styled.div`
+  align-items: start;
+  border-right: 1px solid #ddd;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  justify-content: center;
+  padding: 20px;
+  padding-right: 30px;
+  width: 50%;
+
+  @media (max-width: 768px) {
+    align-items: center;
+    border-bottom: 1px solid #ddd;
+    border-right: 0;
+    border-top: 1px solid #ddd;
+    padding: 30px 20px 30px 20px;
+    width: 100%;
+  }
+`
+const ItineraryDetails = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+  padding-left: 30px;
+
+  @media (max-width: 768px) {
+    padding-top: 30px;
+    width: 100%;
+  }
+`
+const TextWIcon = styled.div`
+  align-items: flex-start;
+  display: flex;
+  gap: 7px;
+  justify-content: left;
+  width: 250px;
+
+  svg {
+    flex-shrink: 0;
+  }
+`
+
+const TripDetailWithIcon = styled(TextWIcon)`
```

Comment:
This thing really has to go. It's duplicated code and there's weird spacing issues. Can we use `MetroItineraryRoutes`?

Reply:
Like I pointed out [here](https://github.com/opentripplanner/otp-react-redux/pull/1151#discussion_r1569013378), this styled component is not creating route blocks! We do use the `MetroItineraryRoutes` in `saved-trip-list`. The `TripDetailWithIcon` is this component: 
![image](https://github.com/opentripplanner/otp-react-redux/assets/115499534/1b1d7e35-04a1-4196-9a8d-4dac0cc80fa3)